release/Deprecation.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
```

Comment:
New deprecation document, theme matches the readme.  It will be in the root of the install.